fuzz/src/bin/msg_channel_ready_target.rs

```diff
@@ -63,11 +63,11 @@ fn run_test_cases() {
 	use std::sync::{atomic, Arc};
 	{
 		let data: Vec<u8> = vec![0];
-		msg_funding_locked_run(data.as_ptr(), data.len());
+		msg_channel_ready_run(data.as_ptr(), data.len());
 	}
 	let mut threads = Vec::new();
 	let threads_running = Arc::new(atomic::AtomicUsize::new(0));
-	if let Ok(tests) = fs::read_dir("test_cases/msg_funding_locked") {
+	if let Ok(tests) = fs::read_dir("test_cases/msg_channel_ready") {
```

Comment:
I'm not sure how large the state is that our fuzzers keep, but if it is large, is there a way to migrate to the renamed test case without discarding the corpus?